### DISCUSSION TITLE
Welcome to Conformer Discussions!

### DISCUSSION THREAD

**Discussion by Rishit-dagli** [Rishit-dagli/Conformer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ntchinda-Giscard** [comment]
Really this library is cool but i can only use a single input shape to  the conformer block that is (1, 1024, 512) if any other input shape is used it throws an error.  Is this a. bug or the library is just meant to receive a particular input shape

**Reply by giridhar-tts** [reply]
Any update on this?
